pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -150,6 +214,24 @@ def conv_block(
 
 
 class JoinBlock(nn.Module):
+    r"""JoinBlock from :cite:`ULVL2016`.
+
+    This block concatenate two inputs along the ``channel _dim`` with previous
+    normalization modules. It therefore corresponds to the ``JoinBlock`` from the paper
+    only without upsample block.
+
+    Args:
+        branch_in_channels: Number of channels in the branch input.
```

Comment:
I wonder if we should rename this to `in_channels` since this block is not restricted to work with branches. Might be good idea for a follow-up PR.